README.md

```diff
@@ -34,6 +34,7 @@ To run unit tests execute the following command:
 Before running the integration tests, set the following environment variables.
 
 *JENKINS_ARTIFACTORY_URL*<br>
```

Comment:
I suggest we retire the JENKINS_ARTIFACTORY_URL and have only JENKINS_PLATFORM_URL for the tests.